data/covering_grammar/lib/make_test_file.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+
+"""Makes test file.
+
+Takes the gold data and the model output, and creates a three-column TSV where
+each line has a word, its gold pronunciation, and the predicted pronunciation.
+Assumes that the input files have the same words in the same order.
+"""
+
+import argparse
+import logging
+
+
+def main(args: argparse.Namespace) -> None:
+    with open(args.gold, "r") as gf, open(args.pred, "r") as pf:
+        with open(args.out, "w") as wf:
+            for i, (g_line, p_line) in enumerate(zip(gf, pf)):
```

Comment:
This line number will be off by one because we start counting as zero. To fix I think you'll do:

`enumerate(zip(gf, pf), 1)`

Also while I'm bikeshedding, `i` is not a great name here. I'd call it `lineno` or something like that?